### PR TITLE
Protocol: attest against EnclaveID rather than HostID

### PR DESCRIPTION
**Pull request by BedrockSquirrel**
### Why this change is needed

Pre-requisite for multi-enclave nodes (to allow enclave failover for sequencer).

### What changes were made as part of this PR

Make enclaveID a first class citizen for attestations.
Disable validator seq signature check until sequencer enclave ID is being captured and circulated to network.

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/ten-protocol/ten-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: contracts/src/management/ManagementContract.sol [hunk 1]
```diff
@@ -23,10 +23,13 @@ contract ManagementContract is Initializable, OwnableUpgradeable {
     // Event to log changes to important contract addresses
     event ImportantContractAddressUpdated(string key, address newAddress);
 
-    mapping(address => string) private attestationRequests;
+    // mapping of enclaveID to attestation request report
+    //mapping(address => string) private attestationRequests; todo: do we need to store this mapping?
```
Comment (BedrockSquirrel) [inline contracts/src/management/ManagementContract.sol:27]: We have been storing the full attestation report in the contract state, but it's not read at the moment. The enclave gets the data from the transaction calldata currently.

Are there reasons it's a good idea to store these attestation reports on the L1?

Comment (tudor-malene) [reply contracts/src/management/ManagementContract.sol:27]: I think it's a vestige from the time we were planning to perform the verification on-chain.
Can be removed

**Review by tudor-malene — COMMENTED** [review]
looks good.
Had a few questions

**Review by tudor-malene — APPROVED** [review]
lgtm

**Pull request merged**